android/src/main/java/com/tailscale/ipn/ui/viewModel/ExitNodePickerViewModel.kt

```diff
@@ -6,23 +6,30 @@ package com.tailscale.ipn.ui.viewModel
 
 import android.util.Log
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.viewModelScope
 import com.tailscale.ipn.ui.localapi.Client
+import com.tailscale.ipn.ui.model.Ipn
 import com.tailscale.ipn.ui.model.StableNodeID
-import com.tailscale.ipn.ui.service.IpnModel
-import com.tailscale.ipn.ui.service.setExitNodeId
 import com.tailscale.ipn.ui.util.LoadingIndicator
 import com.tailscale.ipn.ui.util.set
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import java.util.TreeMap
 
-class ExitNodePickerViewModel(private val model: IpnModel, private val onNavigateHome: () -> Unit) :
-    ViewModel() {
-    companion object {
-        const val TAG = "ExitNodePickerViewModel"
+data class ExitNodePickerNav(
```

Comment:
This now follows the pattern that other models already followed w.r.t. navigation.